src/ignoreFileHandler.ts

```diff
@@ -22,7 +22,9 @@ function ignoreFileHandler(disposables: Disposable[]) {
     return {
         fileIsIgnored(filePath: string) {
             const { ignorer, ignoreFilePath } = getIgnorerForFile(filePath);
-            return ignorer.ignores(path.relative(ignoreFilePath, filePath));
+            return ignorer.ignores(
+                path.relative(path.resolve(ignoreFilePath, '..'), filePath)
```

Comment:
Maybe use `dirname` instead?

Reply:
Ow yep, didn't remember about that function. Certainly nicer